isso/js/tests/unit/isso.test.js

```diff
@@ -1,60 +0,0 @@
-/**
- * @jest-environment jsdom
- */
-
-/* Keep the above exactly as-is!
- * https://jestjs.io/docs/configuration#testenvironment-string
- * https://jestjs.io/docs/configuration#testenvironmentoptions-object
- */
-
-"use strict";
-
-/*
- * Test goals:
- * - Test editorify()
- * - Test insert()
- * - Test insert_loader()
- * - Test Postbox()
- * Also, untangle Postbox functions from DOM element
- */
-
-test('Editorify text area', () => {
```

Comment:
Please instead `test.skip()` this for now. We'll want to test the whole widget at some point in the future again.